src/lib-components/layout/SidebarLayout.vue

```diff
@@ -18,20 +18,20 @@ import { ref } from "vue"
 
 const props = withDefaults(
   defineProps<{
-    activeURL?: string
-    iconURL: string
+    activeUrl?: string
```

Comment:
Nitpick: go is telling us to do activeURL or userID. Specify a rule to keep the ts/go friendship? Not a huge deal..

Reply:
As far as these specific app layout components go, I'd probably just suggest we plan to change the prop names.  The icon prop can probably move to `iconSrc` and I'm not sure if we should have an `activeURL` prop at all.  Will it ever be anything other than what can be read from window.location?

Reply:
No prob, not worth an ongoing battle

Reply:
Yeah better to use window.location for this